### ISSUE TITLE
Marshmallow rule not implemented for Witch and Jester

### ISSUE THREAD

**Issue by ncrecc** [TerryCavanagh/diceydungeons.com#2170]
Is this is already being worked on for v1.10? Not a huge thing but there are three cards entirely inaccessible at the moment due to it (Snow Blast, Flame Shatter, Uncle)

**Comment by TerryCavanagh** [comment]
Nope, not yet - probably a v1.11 thing (but that's because I'm doing v1.10 sooner than originally planned)
